skimage/feature/_texture.pyx

```diff
@@ -12,24 +12,38 @@ from .._shared.transform cimport integrate
 cdef extern from "numpy/npy_math.h":
     double NAN "NPY_NAN"
 
-
-def _glcm_loop(cnp.uint8_t[:, ::1] image, double[:] distances,
+ctypedef fused any_int:
```

Comment:
Would cython.integral suffice here?

See cython/docs/build/html/src/userguide/fusedtypes.html (sorry, the Cython docs are down right now, so I cloned the repo and found the documentation there).